daemon/api_validate.go

```diff
@@ -122,11 +122,23 @@ func listValidationSets(c *Command, r *http.Request, _ *auth.UserState) Response
 	}
```

Comment:
validationSetAccountAndName  is defined in this file but doesn't seem to be used atm?

Reply:
Indeed, it was a leftover from the time the protocol used a single value for account/name and needed splitting, this later changed to separate fields. Removed.